corehq/apps/sms/tasks.py

```diff
@@ -67,11 +67,14 @@ def remove_from_queue(queued_sms):
 
     sms.publish_change()
 
+    tags = {'backend': sms.backend_api}
+    if isinstance(sms.custom_metadata, dict):
+        tags.update(sms.custom_metadata)
```

Comment:
This won't work well with Prometheus which requires that metrics have a consistent set of tags at all times. If there are specific keys you can pull those out and make sure that they get a blank value if they don't exist

Reply:
thanks, updated in 02a713fdfcc